src/application_protocol/services/read_range.rs

```diff
@@ -330,35 +330,35 @@ impl<'a> ReadRangeItem<'a> {
             reader,
             buf,
             TagNumber::ContextSpecificOpening(Self::DATE_TIME_TAG),
-            "ReadRangeItem decode",
+            "ReadRangeItem decode open date time",
```

Comment:
let me know if I should back these changes out - but it made it much easier to debug.

Reply:
They are a little redundant because you have access to the Tag there but if they made your life easier then I have no problem at all putting them in. They are there for debugging after all.